public/components/integrations/components/add_integration_modal.tsx

```diff
@@ -0,0 +1,179 @@
+/*
```

Comment:
Dead code?
If not dead code, there are many questions.

Reply:
Yes dead code, will remove